translations/de.ts

```diff
@@ -1431,17 +1435,17 @@ will be sent to them when they appear online to you.</source>
     <message>
         <location filename="../src/widget/groupwidget.cpp" line="92"/>
         <source>%1 users in chat</source>
-        <translation>%1 Kontakte im Chat</translation>
+        <translation>%1 Benutzer im Chat</translation>
     </message>
     <message>
         <location filename="../src/widget/groupwidget.cpp" line="94"/>
         <source>0 users in chat</source>
-        <translation>Keine Kontakte im Chat</translation>
+        <translation>Keine Benutzer im Chat</translation>
```

Comment:
Einserseits das Original, andererseits, wie gesagt, sind für mich Kontakte die in meiner Kontakliste (oft auch Freunde genannt, was ich nicht mag. Warum? Wenn ich tox nur privat nutze könnte das ja noch hinhauen, aber selbst da: nicht jeder Bekannte ist auch ein Freund. Und wenn ich es beruflich nutze, muss ich dann alles in Kollegen umbenenne? nein. Kontakte ist das richtige Wort für die in in meiner Liste. Aber um die geht es hier ja nicht. Hier geht es um einen Gruppenchat. Der kann gänzlich aus Kontakten bestehten, muss er aber nicht. Es gibt ja auch öffentliche Gruppen. Natürlich stehe ich über die Gruppe mit denen auch in Kontakt, aber wir wollen die beiden Arten von Benutzern doch irgendwie unterscheiden, oder?